package.json

```diff
@@ -28,26 +28,13 @@
       "type": "object",
       "title": "GitHub configuration",
       "properties": {
-        "github.username": {
-          "type": [
-            "string",
-            "null"
-          ],
-          "default": null,
-          "description": "The username to use when accessing GitHub. The default is to consult the Git credential manager."
-        },
-        "github.host": {
-          "type": "string",
-          "default": "github.com",
-          "description": "The host name to access GitHub. Change this to your GitHub Enterprise host."
-        },
-        "github.accessToken": {
-          "type": [
-            "string",
-            "null"
-          ],
-          "default": null,
-          "description": "GitHub access token."
+        "github.hosts": {
+          "type": "array",
+          "default": [],
+          "description": "Host tokens",
+          "items": {
+            "type": "object"
```

Comment:
I think we can also add a `"properties"` property within items, which helps intellisense in the settings editor

something like
```ts
"properties": {
	"host": {
		"type": "string",
		"description": "The host name to access GitHub."
	},
	"username": {
		"type": "string",
		"description": "The host name to access GitHub."
	},
	"token": {
		"type": "string",
		"description": "GitHub access token."
	}
}
```

Reply:
Oh cool, didn't know that, thanks!